glide-core/telemetry/src/lib.rs

```diff
@@ -0,0 +1,68 @@
+use lazy_static::lazy_static;
+use serde::Serialize;
+use std::sync::RwLock as StdRwLock;
+
+#[derive(Default, Serialize)]
+#[allow(dead_code)]
+pub struct Telemetry {
+    /// Total number of connections opened to Valkey
+    total_connections: usize,
+    /// Total number of GLIDE clients
+    total_clients: usize,
```

Comment:
Have you though to separate between CME and standalone client/connection count?

Reply:
There are actually many more custom counters that we can collect (e.g. replicas vs primary, management connections vs user connections etc).

But I want to focus on implementing the feature request as posted by the OP. 
For more complex counters - we will implement this using OpenTelemetry